### DISCUSSION TITLE
wget or download pcap file

### DISCUSSION THREAD

**Discussion by EngineerSuhail** [Security-Onion-Solutions/securityonion#10814] [Unsupported Versions]
"Is it better to download a pcab file to security onion or obtain it through the internet using wget command, or is there no difference?"

**Comment by dougburks** [comment]
A pcap file is a pcap file and it doesn't matter whether it is transported via wget, curl, scp, rsync, or other method.

**Reply by EngineerSuhail** [reply]
I ask because I import PCAP file it affects(appears) in Kibana,  capME and Wireshark but it didnot in sguil I cannot know why?

**Reply by dougburks** [reply]
If you're using Sguil, then that means that you've got an old version of Security Onion that is no longer supported:
https://docs.securityonion.net/en/2.3/eol.html
